pkg/fuzzer/queue/avoid.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package queue
+
+import (
+	"sync"
+	"sync/atomic"
+
+	"github.com/google/syzkaller/pkg/stats"
+)
+
+type Avoider struct {
```

Comment:
Nit: this is probably a `Distributor`? :) Eventually we'll have a crashing syscalls avoider here and it will be quite confusing.

Reply:
Done